vendor/github.com/containerd/go-runc/runc.go

```diff
@@ -750,7 +750,6 @@ func parseVersion(data []byte) (Version, error) {
 // Availability:
 //
 //   - runc:  supported since runc v1.1.0
-//   - crun:  https://github.com/containers/crun/issues/1177
```

Comment:
Should be like `- crun: supported since crun vX.X.X`

Reply:
Done in https://github.com/containerd/go-runc/pull/103